ui/views.py

```diff
@@ -67,6 +84,8 @@ def get_context_data(self, **kwargs):
         video = context["object"]
         refresh_status(video)
         context['form'] = VideoForm(instance=video)
+        context['videofile'] = context['object'].videofile_set.filter(encoding=EncodingNames.HLS)
+        context["js_settings_json"] = default_js_settings(self.request)
```

Comment:
This doesn't seem to be set on the login page. Can you investigate how to do so?